modules/persistent-storage-csi-sc-multiple-none.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-sc-manage
+//
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-sc-multiple-none_{context}"]
+= Absent or multiple default storage classes
+
+== Multiple default storage classes
+Multiple default storage classes can occur if you mark a non-default storage class as default and do not unset the existing default storage class, or you create a default storage class when a default storage class is already present. With multiple default storage classes present, any persistent volume claim (PVC) requesting the default storage class (`pvc.spec.storageClassName`=nil) gets the most recently created default storage class, and the administrator receives an alert that there are multiple default storage classes.
+
```

Comment:
@RomanBednar Hi could you tell me where I can see the alert that there are multiple default storage classes?

Reply:
@chao007 you'll find it alerts dashboard just like any other alert, if it's triggered. It's called `MultipleDefaultStorageClasses` and was added [here](https://github.com/openshift/cluster-storage-operator/pull/276/commits/57f7a060abb4e2a7a6353665544b3b310c91621e#diff-9765db0de5fb4e276380f3da0ebffdecc5501c0db232c911d40fa3ee92980939R16).